tools/SetupFlow/DevHome.SetupFlow/ViewModels/Environments/EnvironmentCreationOptionsViewModel.cs

```diff
@@ -234,15 +238,14 @@ public void OnAdaptiveCardUpdated(object sender, AdaptiveCard adaptiveCard)
             // Render the adaptive card and set the action event handler.
             _renderedAdaptiveCard = _adaptiveCardRenderer.RenderAdaptiveCard(adaptiveCard);
             _renderedAdaptiveCard.Action += OnRenderedAdaptiveCardAction;
-
-            // Send new card to listeners
             _userInputsFromAdaptiveCard = _renderedAdaptiveCard.UserInputs;
-            WeakReferenceMessenger.Default.Send(new NewAdaptiveCardAvailableMessage(new RenderedAdaptiveCardData(Orchestrator.CurrentPageViewModel, _renderedAdaptiveCard)));
-            IsAdaptiveCardSessionLoaded = true;
 
-            // We set CanGoToNextPage to true here because we can only validate the inputs when the user interacts with the adaptive card
-            // via the action buttons.
-            CanGoToNextPage = true;
+            if (_isFirstTimeLoadingNewFlow)
```

Comment:
Is this so that we don't refresh the page? Consider adding a comment.

Reply:
yea I can add a comment